WorldBank-Transport/DRIVER
# Allow filtering records by creator

## Overview
Adds a text filter field so that a user can view only records that were created by a particular user, using either their username or email address.

### Checklist


### Demo
![screen shot 2018-09-10 at 3 27 43 pm](https://user-images.githubusercontent.com/1032849/45319551-161d7900-b50e-11e8-81a1-8efeb7eb8d06.png)

## Testing Instructions
- Create some records
- Using the "Created by" filter box, filter by your user's username
  - You should see the records you created
- Filter by another string
  - You should not see the records you created
- (Hinging on [PT160399313](https://www.pivotaltracker.com/story/show/160399313)) Log out to see the Record filterbar
  - You should see the keyword, created date, and occurred date filters, but _not_ created by

Closes [PT159629145](https://www.pivotaltracker.com/story/show/159629145)
